### PR TITLE
add course

### PR DESCRIPTION
**Pull request by wowKimsohyun17**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/30221)

## WHAT this PR does?
Admin create course

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by wowKimsohyun17** [comment]
ready

File: app/Http/Controllers/Admin/CourseController.php [hunk 1]
```diff
@@ -16,7 +16,7 @@ class CourseController extends Controller
      */
     public function index()
     {
-        $courses = Course::all();
+        $courses = Course::with('image')->paginate(3);
```
Comment (huyenttm-0901) [inline app/Http/Controllers/Admin/CourseController.php]: đưa số vào config

File: resources/views/admin/component/add_course.blade.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                                                @enderror
+                                            </div>
+                                            <div class="form-group">
+                                                <input type='file' id="input-img" name="photo"/>
+                                                <img id="img-preview" src="#" class="d-none"/>
+                                                @error('photo')
+                                                    <p class="error-message">{{ $message }}</p>
+                                                @enderror
+                                            </div>
+                                            <button type="submit" class="btn btn-primary">Submit</button>
```
Comment (huyenttm-0901) [inline resources/views/admin/component/add_course.blade.php]: i18n

**Review by huyenttm-0901 — COMMENTED** [review]
thêm ticket redmine

**Comment by wowKimsohyun17** [comment]
ready

**Pull request merged**